github/ci/prow/files/jobs/kubevirt/user-guide/user-guide-postsubmits.yaml

```diff
@@ -0,0 +1,47 @@
+postsubmits:
+  kubevirt/user-guide:
+  - name: push-user-guide-source-build-and-push-to-gh-pages
+    branches:
+    - ^source$
+    annotations:
+      testgrid-create-test-group: "false"
+    decorate: true
+    extra_refs:
+    - org: kubevirt
+      repo: user-guide
+      base_ref: gh-pages
+      path_alias: github.com/kubevirt/user-guide-gh-pages
+    - org: kubevirt
+      repo: project-infra
+      base_ref: master
+    spec:
+      containers:
+      - image: docker.io/library/python:3.7
+        env:
+        - name: GIT_ASKPASS
+          value: ../project-infra/hack/git-askpass.sh
+        - name: GIT_AUTHOR_NAME
+          value: kubevirt-bot
+        - name: GIT_AUTHOR_EMAIL
+          value: rmohr+kubebot@redhat.com
+        command: [ "/bin/sh", "-c" ]
+        args:
+        - |
+          set -e
+          git config --global user.email "$GIT_AUTHOR_EMAIL"
+          git config --global user.name "$GIT_AUTHOR_NAME"
+          commit=$(git show -s --format=%H)
+          make build
+          cd ../user-guide-gh-pages
+          find . -mindepth 1 -maxdepth 1 -regextype egrep -not -regex '\./(\.git.*|\.nojekyll)' -print | xargs rm -rf
+          cp -R ../user-guide/site/* .
+          git add -A
+          git commit --signoff -m "Postsubmit site update from $commit"
+          git push "https://kubevirt-bot@github.com/kubevirt/user-guide.git" HEAD:gh-pages
```

Comment:
Thank you for writing this.

I don't see a `mkdir` command.  Is directory `user-guide-gh-pages` going to exist when this script runs?

Reply:
The section below extra_refs [here](https://github.com/kubevirt/project-infra/pull/909/files#diff-4da29374e94c42b91b099fa28d3658e9899839ccd6133122b4d57aff1f8543e8R10-R13) takes care of checking out our base for updating the gh-pages branch, see [extra_refs docs here](https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md#what-the-test-container-can-expect)